.github/workflows/elixir.yml

```diff
@@ -0,0 +1,21 @@
+name: Elixir CI
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: elixir:1.9.1-slim
```

Comment:
Note that we use Elixir 1.9.4 / Erlang/OTP 22 in dev, so the same environment should be used in test

Reply:
I see, I will try implement these changes, but you know your build environment best.
I never used Elixir before, I am mostly using Actions for DotNetCore.

Reply:
Given that I have never run the tests before, I do not expect them to pass, but it's certainly a welcome addition :slightly_smiling_face:

Reply:
01:07:02.160 [error] GenServer #PID<0.6672.0> terminating
** (DBConnection.ConnectionError) tcp connect (localhost:5432): connection refused - :econnrefused
    (db_connection) lib/db_connection/connection.ex:87: DBConnection.Connection.connect/2
    (connection) lib/connection.ex:622: Connection.enter_connect/5
    (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Last message: nil
** (Mix) The database for Philomena.Repo couldn't be created: killed
##[error]Process completed with exit code 1.

Otherwise latest commit changes made test get a lot more far.

Reply:
I recommend committing and taking every test fail as motivation to fix it.

Reply:
You need a database server (postgresql) to be running and available on localhost with the following config
https://github.com/derpibooru/philomena/blob/c8e63adc22e51e160f3d7043c4cb6789296c92ff/config/test.exs#L4-L8

Reply:
Working on it